### DISCUSSION TITLE
Failed to Download

### DISCUSSION THREAD

**Discussion by shuyuewang27** [aclap-dev/video-downloadhelper#759] [Help and Q&A]
### Your question

Hi all, I failed to download the videos. Can someone help? 

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://kev2.kmf.com/ke-video/#/replay/985962/1",
  "error": {
    "details": "SideDownload error: : [hls @ 0x131f0b230] Unable to open key file https://hls.videocc.net/510de8f5bb/e/510de8f5bb3be96c7b90f692fcc27f7e_2.key\n[hls @ 0x131f0b230] Error when loading first segment 'https://hw-dts.videocc.net/510de8f5bb/0/1716731094135/e/c2/7f/7e_2/510de8f5bb3be96c7b90f692fcc27f7e_2_0.ts?pid=1716964000077X1496265&device=desktop'\n[in#0 @ 0x131f0b130] Error opening input: Invalid data found when processing input\nError opening input file https://hls.videocc.net/510de8f5bb/e/510de8f5bb3be96c7b90f692fcc27f7e_2.m3u8?pid=1716964000077X1496265&device=desktop.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_5662291320907397",
    "error": "coapp_failure",
    "id": "coapp_failure_a523593e-e5f6-40a2-916b-3d6bbfd9f7e9"
  }
}

**Comment by Mattmjs** [comment]
Unable to open key file.
